packages/driver/cypress/e2e/e2e/origin/cookie_login.cy.ts

```diff
@@ -191,7 +191,8 @@ describe('cy.origin - cookie login', () => {
       verifyIdpNotLoggedIn({ expectNullCookie: false })
     })
 
-    it('SameSite=None -> not logged in', () => {
+    // FIXME: Currently in Firefox, the default cookie setting in the extension is no_restriction, which can be set with Secure=false.
+    it('SameSite=None -> not logged in', { browser: '!firefox' }, () => {
```

Comment:
is this broken or the tests arn't aligned? should we add a test specifically for firefox?

Reply:
What I believe was happening here before is that since this request that sets cookies was seemed "cross-origin", `X-Set-Cookie` was being used in the header and not setting the cookie in the browser. In the server side cookie jar, we [ignore cookies](https://github.com/cypress-io/cypress/blob/develop/packages/proxy/lib/http/util/cookies.ts#L114) that are `SameSite=None` and `Secure=false`.

Now, since we use `Set-Cookie` all the time, the cookie is actually set in firefox because we currently allow `SameSite=None` and `Secure=false` to be set in the extension, which actually allows this test to log in.

We can add a test that verifies this behavior, but I am somewhat of the opinion that we shouldn't allow setting insecure samesite none cookies in firefox.

Reply:
> Now, since we use Set-Cookie all the time, the cookie is actually set in firefox because we currently allow SameSite=None and Secure=false to be set in the extension, which actually allows this test to log in.

I would expect this to be a problem for our user if it works in chrome one way and differently in firefox. Shouldn't we be updating the extension to have the same checks / ignore the same attributes as the server-side cookie jar that is using to manage the cookies for chrome & electron ?